lib/src/realm_class.dart

```diff
@@ -62,6 +63,11 @@ class Realm {
     }
   }
 
+  static void deleteRealm(Configuration config) {
```

Comment:
Should we spawn an isolate and make this async?

Reply:
I don't think we should try doing that. This should be a sync operation as all other core related.

Reply:
Okay, I just don't like blocking IO on the main thread.

Reply:
After a discussion we decided to have only one sync method deleteReam for now.